app/helpers/dashboard_helper.rb

```diff
@@ -15,6 +15,8 @@ def dashboard_actions
 
   def render_widget(widget)
     render(:partial => widget.template, :locals => widget.data)
+  rescue ActionView::MissingTemplate
+    _("Missing template '%{template}' for widget '%{widget}'. The widget or a plugin providing it was probably removed") % { :widget => widget.name, :template => widget.template }
```

Comment:
Can you raise a `Foreman::Exception` that is shown in an alert here instead? I think we can do a better job explaining possible causes by filling the wiki for a given ERF rather than just mention a cause (without the solution) in the message itself.